tests/sub_port_interfaces/sub_ports_helpers.py

```diff
@@ -110,3 +250,26 @@ def get_port_number(interface):
     Returns: Number of port
     """
     return ''.join([i for i in interface.split('.')[0] if i.isdigit()])
+
+
+def get_port_mtu(duthost, interface):
+    """
+    Get MTU of port from interface name
+
+    Args:
+        duthost: DUT host object
+        interface: Full interface name
+
+    Returns: MTU
+    """
+    pattern = ''
+    out = ''
+
+    if '.' in interface:
+        pattern = re.compile(r'%s\s+\S+\s+(\d+)' % interface)
+        out = duthost.shell("show subinterface status {}".format(interface))["stdout"]
+    else:
+        pattern = re.compile(r'%s\s+\S+\s+\S+\s+(\d+)' % interface)
+        out = duthost.shell("show interface status {}".format(interface))["stdout"]
```

Comment:
The output of `show interface status` is tabulate format. It would be easier to use `duthost.show_and_parse` method than regex.

https://github.com/Azure/sonic-mgmt/blob/master/tests/common/devices.py#L1026